src/frontend/cypress/e2e/header.cy.js

```diff
@@ -0,0 +1,96 @@
+beforeEach(() => {
+  cy.login({ user: "user1", password: "password1" });
+});
+
+describe("header component spec", () => {
+  it("header component exists after login", () => {
+    cy.litElementExist("il-app,il-chat,il-chat-header");
```

Comment:
Visto che il path ai componenti è scritto uguale in più punti magari metti una costante che lo contiene (lo stesso anche per eventuali altre stringhe di duplicazione).
Dove c'è questo path + un altro pezzo aggiungilo con la string interpolation. (Se è utilizzato in un solo punto va bene farla lì, se no crea un'altra costante)